packages/sdk/src/util/api-model.ts

```diff
@@ -0,0 +1,8 @@
+// For VC SDK
```

Comment:
Could you please rename this file to `api-model-types.ts`?

Reply:
how about `sdk-api-types.ts`

Reply:
have updated to `sdk-api-types.ts`